Replace proxy #10521 ke (#130)
uyuni-project/uyuni-docs: modules/installation/pages/uyuni-proxy-setup.adoc

== Install the [path]``uyuni_proxy`` pattern

Check that the Proxy pattern is installed correctly.

> On the server select the [package]``pattern_uyuni_proxy`` package for installation, or make sure the [path]``uyuni_proxy`` pattern is installed using the following command on the proxy as root:

Reviewer: ```suggestion
On the server select the [package]``pattern_uyuni_proxy`` package for installation, or make sure the [path]``uyuni_proxy`` pattern is installed using this command on the proxy as root:
```

----

[[replace-uyuniproxy]]

> == Replace a Uyuni Proxy

Reviewer 2: `{productname} Proxy` maybe?

A proxy does not contain any information about the clients that are connected to it.
Therefore, a proxy can be replaced by a new one at any time.
The replacement proxy must have the same name and IP address as its predecessor.

> Shut down the old proxy, but leave it installed while you prepare the replacement.

Reviewer: ```suggestion
Shut down the old proxy, and leave it installed while you prepare the replacement.
```

Create a reactivation key for this system and then register the new proxy using the reactivation key.
If you do not use the reactivation key, you will need to re-register all the

> clients against the new proxy.

Reviewer: ```suggestion
If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.
```

[NOTE]
====
The reactivation key is only needed if you do not want to lose the history of the machine.
If you do not use a reactivation key, the replacement proxy will become a "new" one with a new ID.
====

> Here is a detailed procedure step-by-step:

Reviewer: ```suggestion
```

[[proxy.migration3.replace]]
.Procedure: Replacing a Proxy and Keeping the Clients Registered
. Before starting the actual migration procedure, save the data from the old proxy, if needed.
Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
. Shut down the proxy.

> . Install a new proxy{nbsp}{productnumber}, following xref:install-proxy-uyuni.adoc[Uyuni Proxy Installation].

Reviewer: ```suggestion
. Install a new {productname} Proxy.
For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
```

----

[[replacing.a.uyuniproxy]]
== Replace a Uyuni Proxy

A proxy does not contain any information about the clients that are connected to it.
Therefore, a proxy can be replaced by a new one at any time.
The replacement proxy must have the same name and IP address as its predecessor.

Leave the old proxy in place while you prepare the replacement.
Create a reactivation key for this system and then register the new proxy using the reactivation key.
If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.

[[proxy.migration3.replace]]
.Procedure: Replacing a Proxy and Keeping the Clients Registered
. Before starting the actual migration procedure, save the data from the old proxy, if needed.
Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
. Shut down the proxy.
. Install a new proxy{nbsp}{productnumber}, following xref:install-proxy-uyuni.adoc[Uyuni Proxy Installation].
. In the {productname} {webui} select the newly installed {susemgrproxy} and delete it from the systems list.

> . In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.

Reviewer 2: @mantel do we need this new reactivation key?

Reviewer 3: The re-activation key is only needed if you do not want to lose the history of the machine. Else it will become a "new" one with a new ID.

Author: added this info also for uyuni
